uilib/src/components/Text/getTextStyle.ts

```diff
@@ -0,0 +1,117 @@
+export type TextTypes =
+  | "h1"
+  | "h2"
+  | "h3"
+  | "highlight"
+  | "emphasis"
+  | "body"
+  | "cta"
+  | "link"
+  | "tiny"
+  | "subTitle"
+  | "navigation"
+  | "tag";
+
+export default function getTextStyle({
+  type,
+  bracket,
+}: {
+  type: TextTypes;
+  bracket?: boolean;
+}): {
+  fontFamily: string;
+  fontSize: number;
+  lineHeight: number;
+  fontWeight?: number;
+  textDecoration?: string;
+  paddingTop?: number;
+} {
+  switch (type) {
+    case "h1":
+      return {
+        fontFamily: "Alpha",
+        fontSize: 36,
```

Comment:
How are we getting the font sizes and line heights for these? They don't match with the Figma's @LFBarreto @juan-cortes @jgiamporcaro-ledger